app/page.tsx

```diff
@@ -9,6 +9,7 @@ const Home = () => {
   const [AILoading, setAILoading] = useState<boolean>(false);
   const [chatActiveId, setChatActiveId] = useState<string>('');
   const [chatList, setChatList] = useState<Chat>({ messages: [] });
+  const [userId] = useState<string>(uuidv4());
```

Comment:
Usually, the use of ```useState``` is to sustain a value that will change during an action and that's how a component know how it have to re-render, that's why in other states we use ```[state, setState]```, since that you alredy don't have a setter for a new state, you can make it more simple using a const variable.